lib/AST/NameLookup.cpp

```diff
@@ -1143,37 +1143,30 @@ void ExtensionDecl::addedMember(Decl *member) {
 // MemberLookupTable is constructed (and possibly has entries in it),
 // MemberLookupTable is incrementally reconstituted with new members.
 
-static bool
+static void
 populateLookupTableEntryFromLazyIDCLoader(ASTContext &ctx,
                                           MemberLookupTable &LookupTable,
                                           DeclBaseName name,
                                           IterableDeclContext *IDC) {
+  assert(!IDC->isLoadingLazyMembers());
```

Comment:
Let's see if your assert can stand up to the bots.  You might be in the same position I am right now with the lookup requests.